log_parser/log.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+
+class Log:
+    """
+    Class representing a log entry with a message and a timestamp.
+    """
+
+    def __init__(self, message: str, timestamp: str):
```

Comment:
If the timestamp is in iso format, maybe we should rename the variable that way? I'm not sure, but when I see a string timestamp I don't know what to expect